<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML>
<HEAD>
	<META HTTP-EQUIV="CONTENT-TYPE" CONTENT="text/html; charset=utf-8">
	<TITLE>CHAPTER 25 </TITLE>
	<META NAME="GENERATOR" CONTENT="LibreOffice 4.1.6.2 (Linux)">
	<META NAME="AUTHOR" CONTENT="pearson education">
	<META NAME="CREATED" CONTENT="20061128;151100000000000">
	<META NAME="CHANGEDBY" CONTENT="cmassengale">
	<META NAME="CHANGED" CONTENT="20061128;151100000000000">
	<META NAME="_AdHocReviewCycleID" CONTENT="851268486">
	<META NAME="_AuthorEmail" CONTENT="Julia.Khait@aw.com">
	<META NAME="_AuthorEmailDisplayName" CONTENT="Khait, Julia">
	<META NAME="_EmailSubject" CONTENT="25_LectureOutline">
	<META NAME="_ReviewingToolsShownOnce" CONTENT="">
	<STYLE TYPE="text/css">
	<!--
		@page { size: 8.5in 11in; margin-right: 1in; margin-top: 1in; margin-bottom: 0.75in }
		P { margin-bottom: 0.08in; direction: ltr; color: #000000; widows: 2; orphans: 2 }
		P.western { font-family: "Times New Roman", serif; font-size: 12pt; so-language: en-US }
		P.cjk { font-family: "Times New Roman", serif; font-size: 12pt }
		P.ctl { font-family: "Times New Roman", serif; font-size: 12pt; so-language: ar-SA }
	-->
	</STYLE>
</HEAD>
<BODY LANG="en-US" TEXT="#000000" DIR="LTR">
<P ALIGN=CENTER STYLE="margin-bottom: 0in"><FONT FACE="Times, serif"><FONT SIZE=4 STYLE="font-size: 16pt"><B><FONT COLOR="#ff0000"><FONT FACE="Comic Sans MS, cursive"><SPAN STYLE="text-decoration: none">Chapter
25   Phylogeny and Systematics</SPAN></FONT></FONT></B></FONT></FONT></P>
<P ALIGN=CENTER STYLE="margin-bottom: 0in"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><I><B>Lecture
Outline</B></I></FONT></FONT></P>
<P STYLE="margin-top: 0.22in; margin-bottom: 0.17in; text-decoration: none">
<FONT COLOR="#ff0000"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>Overview:
Investigating the Tree of Life</B></FONT></FONT></FONT></P>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Evolutionary
	biology is about both process and history.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The processes of
	evolution are natural selection and other mechanisms that change the
	genetic composition of populations and can lead to the evolution of
	new species.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>A major goal of
	evolutionary biology is to reconstruct the history of life on earth.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>In
	this chapter, we will consider how scientists trace </FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>phylogeny,</B></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>
	the evolutionary history of a group of organisms.</FONT></FONT></FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>To
	reconstruct phylogeny, scientists use </FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>systematics,</B></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>
	an analytical approach to understanding the diversity and
	relationships of living and extinct organisms.</FONT></FONT></FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Evidence used to
	reconstruct phylogenies can be obtained from the fossil record and
	from morphological and biochemical similarities between organisms.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>In
	recent decades, systematists have gained a powerful new tool in
	</FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>molecular
	systematics,</B></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>
	which uses comparisons of nucleotide sequences in DNA and RNA to
	help identify evolutionary relationships between individual genes or
	even entire genomes.</FONT></FONT></FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Scientists are
	working to construct a universal tree of life, which will be refined
	as the database of DNA and RNA sequences grows.</FONT></FONT></P>
</UL>
<P STYLE="margin-top: 0.17in; margin-bottom: 0in; line-height: 0.17in; text-decoration: none">
<BR>
</P>
<P STYLE="margin-top: 0.17in; margin-bottom: 0in; line-height: 0.17in; text-decoration: none">
<FONT COLOR="#ff0000"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>Concept
25.1 Phylogenies are based on common ancestries inferred from fossil,
morphological, and molecular evidence</B></FONT></FONT></FONT></P>
<P STYLE="margin-top: 0.13in; margin-bottom: 0in; line-height: 0.17in">
 <FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><I><B>Sedimentary
rocks are the richest source of fossils.</B></I></FONT></FONT></P>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Fossils are the
	preserved remnants or impressions left by organisms that lived in
	the past.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>In essence, they
	are the historical documents of biology.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Sedimentary rocks
	form from layers of sand and silt that are carried by rivers to seas
	and swamps, where the minerals settle to the bottom along with the
	remains of organisms.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>As deposits pile
	up, they compress older sediments below them into layers called
	strata.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The
	</FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>fossil
	record</B></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>
	is the ordered array in which fossils appear within sedimentary rock
	strata.</FONT></FONT></FONT></FONT></P>
</UL>
<P STYLE="margin-left: 0.75in; text-indent: -0.25in; margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>These rocks record
the passing of geological time.</FONT></FONT></P>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Fossils can be used
	to construct phylogenies only if we can determine their ages.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The fossil record
	is a substantial, but incomplete, chronicle of evolutionary change.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The majority of
	living things were not captured as fossils upon their death.</FONT></FONT></P>
</UL>
<P STYLE="margin-left: 0.75in; text-indent: -0.25in; margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Of those that formed
fossils, later geological processes destroyed many.</FONT></FONT></P>
<P STYLE="margin-left: 0.75in; text-indent: -0.25in; margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Only a fraction of
existing fossils have been discovered.</FONT></FONT></P>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The fossil record
	is biased in favor of species that existed for a long time, were
	abundant and widespread, and had hard shells or skeletons that
	fossilized readily.</FONT></FONT></P>
</UL>
<P STYLE="margin-top: 0.13in; margin-bottom: 0in; line-height: 0.17in">
 <FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><I><B>Morphological
and molecular similarities may provide clues to phylogeny.</B></I></FONT></FONT></P>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Similarities due to
	shared ancestry are called homologies.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Organisms that
	share similar morphologies or DNA sequences are likely to be more
	closely related than organisms without such similarities.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Morphological
	divergence between closely related species can be small or great.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Morphological
	diversity may be controlled by relatively few genetic differences.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Similarity
	due to convergent evolution is called </FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>analogy.</B></FONT></FONT></FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>When two organisms
	from different evolutionary lineages experience similar
	environmental pressures, natural selection may result in convergent
	evolution.</FONT></FONT></P>
</UL>
<P STYLE="margin-left: 0.75in; text-indent: -0.25in; margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Similar analogous
adaptations may evolve in such organisms.</FONT></FONT></P>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Analogies are not
	due to shared ancestry.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Distinguishing
	homology from analogy is critical in the reconstruction of
	phylogeny.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>For example, both
	birds and bats have adaptations that allow them to fly.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>However, a close
	examination of a bat’s wing shows a greater similarity to a cat’s
	forelimb that to a bird’s wing.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Fossil evidence
	also documents that bat and bird wings arose independently from
	walking forelimbs of different ancestors.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Thus
	a bat’s wing is </FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><I>homologous</I></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>
	to other mammalian forelimbs but is </FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><I>analogous</I></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>
	in function to a bird’s wing.</FONT></FONT></FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Analogous
	structures that have evolved independently are also called
	</FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>homoplasies.</B></FONT></FONT></FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>In general, the
	more points of resemblance that two complex structures have, the
	less likely it is that they evolved independently.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>For example, the
	skulls of a human and a chimpanzee are formed by the fusion of many
	bones.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The two skulls
	match almost perfectly, bone for bone.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>It is highly
	unlikely that such complex structures have separate origins.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>More likely, the
	genes involved in the development of both skulls were inherited from
	a common ancestor.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The same argument
	applies to comparing genes, which are sequences of nucleotides.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>Systematists</B></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>
	compare long stretches of DNA and even entire genomes to assess
	relationships between species.</FONT></FONT></FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>If genes in two
	organisms have closely similar nucleotide sequences, it is highly
	likely that the genes are homologous.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>It may be difficult
	to carry out molecular comparisons of nucleic acids.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The first step is
	to align nucleic acid sequences from the two species being studied.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>In closely related
	species, sequences may differ at only one or a few sites.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Distantly related
	species may have many differences or sequences of different length.</FONT></FONT></P>
</UL>
<P STYLE="margin-left: 0.75in; text-indent: -0.25in; margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Over evolutionary
time, insertions and deletions accumulate, altering the lengths of
the gene sequences.</FONT></FONT></P>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>Deletions
	or insertions</B></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>
	may shift the remaining sequences, making it difficult to recognize
	closely matching nucleotide sequences.</FONT></FONT></FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>To deal with this,
	systematists use computer programs to analyze comparable DNA
	sequences of differing lengths and align them appropriately.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The fact that
	molecules have diverged between species does not tell us how long
	ago their common ancestor lived.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Molecular
	divergences between lineages with reasonably complete fossil records
	can serve as a molecular yardstick to measure the appropriate time
	span of various degrees of divergence.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>As
	with morphological characters, it is necessary to distinguish
	</FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>homology
	from analogy</B></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>
	to determine the usefulness of molecular similarities for
	reconstruction of phylogenies.</FONT></FONT></FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Closely similar
	sequences are most likely homologies.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>In distantly
	related organisms, identical bases in otherwise different sequences
	may simply be coincidental matches or molecular homoplasies.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Scientists
	have developed </FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>mathematical
	tools </B></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>that
	can distinguish “distant” homologies from coincidental matches
	in extremely divergent sequences.</FONT></FONT></FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>For example, such
	molecular analysis has provided evidence that humans share a distant
	common ancestor with bacteria.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Scientists have
	sequenced more than 20 billion bases worth of nucleic acid data from
	thousands of species.</FONT></FONT></P>
</UL>
<P STYLE="margin-top: 0.17in; margin-bottom: 0in; line-height: 0.17in; text-decoration: none">
<BR>
</P>
<P STYLE="margin-top: 0.17in; margin-bottom: 0in; line-height: 0.17in; text-decoration: none">
<BR>
</P>
<P STYLE="margin-top: 0.17in; margin-bottom: 0in; line-height: 0.17in; text-decoration: none">
<FONT COLOR="#ff0000"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>Concept
25.2 Phylogenetic systematics connects classification with
evolutionary history</B></FONT></FONT></FONT></P>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>In
	1748, Carolus Linnaeus published </FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><I>Systema
	naturae</I></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>,
	his classification of all plants and animals known at the time.</FONT></FONT></FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Taxonomy is an
	ordered division of organisms into categories based on similarities
	and differences.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Linneaus’s
	classification was not based on evolutionary relationships but
	simply on resemblances between organisms.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Despite this, many
	features of his system remain useful in phylogenetic systematics.</FONT></FONT></P>
</UL>
<P STYLE="margin-top: 0.13in; margin-bottom: 0in; line-height: 0.17in">
 <FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><I><B>Taxonomy
employs a hierarchical system of classification.</B></I></FONT></FONT></P>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The
	Linnaean system, first formally proposed by Linnaeus in </FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><I>Systema
	naturae</I></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>
	in the 18th century, has two main characteristics.</FONT></FONT></FONT></FONT></P>
</UL>
<OL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Each species has a
	two-part name.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Species are
	organized hierarchically into broader and broader groups of
	organisms.</FONT></FONT></P>
</OL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Under
	the binomial system, each species is assigned a two-part Latinized
	name, a </FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>binomial.</B></FONT></FONT></FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The
	first part, the </FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>genus,</B></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>
	is the closest group to which a species belongs.</FONT></FONT></FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The
	second part, the </FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>specific
	epithet,</B></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>
	refers to one </FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>species</B></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>
	within each genus.</FONT></FONT></FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The first letter of
	the genus is capitalized and both names are italicized and
	Latinized.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>For
	example, Linnaeus assigned to humans the optimistic scientific name
	</FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><I>Homo
	sapiens</I></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>,
	which means “wise man.”</FONT></FONT></FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>A
	</FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>hierarchical
	classification</B></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>
	groups species into increasingly broad taxonomic categories.</FONT></FONT></FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Species that appear
	to be closely related are grouped into the same genus.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>For
	example, the leopard, </FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><I>Panthera
	pardus</I></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>,
	belongs to a genus that includes the African lion (</FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><I>Panthera
	leo</I></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>)
	and the tiger (</FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><I>Panthera
	tigris</I></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>).</FONT></FONT></FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Genera
	are grouped into progressively broader categories: </FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>family,
	order, class, phylum, kingdom,</B></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>
	and </FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>domain.</B></FONT></FONT></FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Each taxonomic
	level is more comprehensive than the previous one.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>As an example, all
	species of cats are mammals, but not all mammals are cats.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The
	named taxonomic unit at any level is called a </FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>taxon.</B></FONT></FONT></FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Example:
	</FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><I>Panthera</I></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>
	is a taxon at the genus level, and Mammalia is a taxon at the class
	level that includes all of the many orders of mammals.</FONT></FONT></FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Higher
	classification levels are not defined by some measurable
	characteristic, such as the reproductive isolation that separates
	biological species.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>As a result, the
	larger categories are not comparable between lineages.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>An order of snails
	does not necessarily exhibit the same degree of morphological or
	genetic diversity as an order of mammals.</FONT></FONT></P>
</UL>
<P STYLE="margin-top: 0.13in; margin-bottom: 0in; line-height: 0.17in">
 <FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><I><B>Classification
and phylogeny are linked.</B></I></FONT></FONT></P>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Systematists
	explore phylogeny by examining various characteristics in living and
	fossil organisms.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>They construct
	branching diagrams called phylogenetic trees to depict their
	hypotheses about evolutionary relationships.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The branching of
	the tree reflects the hierarchical classification of groups nested
	within more inclusive groups.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Methods for tracing
	phylogeny began with Darwin, who realized the evolutionary
	implications of Linnaean hierarchy.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Darwin
	introduced phylogenetic systematics in </FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><I>On
	the Origin of Species</I></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>
	when he wrote: “Our classifications will come to be, as far as
	they can be so made, genealogies.”</FONT></FONT></FONT></FONT></P>
</UL>
<P STYLE="margin-top: 0.13in; margin-bottom: 0in; font-style: normal; line-height: 0.17in">
<BR>
</P>
<P STYLE="margin-top: 0.13in; margin-bottom: 0in; font-style: normal; line-height: 0.17in">
<FONT COLOR="#ff0000"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>Concept
25.3 Phylogenetic systematics informs the construction of
phylogenetic trees based on shared characters</B></FONT></FONT></FONT></P>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Patterns
	of shared characteristics can be depicted in a diagram called a
	</FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>cladogram</B></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>.</FONT></FONT></FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>If shared
	characteristics are homologous and, thus, explained by common
	ancestry, then the cladogram forms the basis of a phylogenetic tree.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>A
	</FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>clade</B></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>
	is defined as a group of species that includes an ancestral species
	and all its descendents.</FONT></FONT></FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The
	study of resemblances among clades is called </FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>cladistics.</B></FONT></FONT></FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Each branch, or
	clade, can be nested within larger clades.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>A
	valid clade is </FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>monophyletic,</B></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>
	consisting of an ancestral species and all its descendents.</FONT></FONT></FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>When
	we lack information about some members of a clade, the result is a
	</FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>paraphyletic</B></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>
	grouping that consists of some, but not all, of the descendents.</FONT></FONT></FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The
	result may also be several </FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>polyphyletic</B></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>
	groupings that lack a common ancestor.</FONT></FONT></FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Such situations
	call for further reconstruction to uncover species that tie these
	groupings together into monophyletic clades.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Determining which
	similarities between species are relevant to grouping the species in
	a clade is a challenge.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>It is especially
	important to distinguish similarities that are based on shared
	ancestry or homology from those that are based on convergent
	evolution or analogy.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Systematists must
	also sort through homologous features, or characters, to separate
	shared derived characters from shared primitive characters.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>A “character”
	refers to any feature that a particular taxon possesses.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>A
	</FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>shared
	derived character</B></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>
	is unique to a particular clade.</FONT></FONT></FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>A
	</FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>shared
	primitive character</B></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>
	is found not only in the clade being analyzed, but also in older
	clades.</FONT></FONT></FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>For example, the
	presence of hair is a good character to distinguish the clade of
	mammals from other tetrapods.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>It is a shared
	derived character that uniquely identifies mammals.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>However, the
	presence of a backbone can qualify as a shared derived character,
	but at a deeper branch point that distinguishes all vertebrates from
	other mammals.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Among vertebrates,
	the backbone is a shared primitive character because it evolved in
	the ancestor common to all vertebrates.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Shared derived
	characters are useful in establishing a phylogeny, but shared
	primitive characters are not.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The status of a
	character shared derived versus shared primitive may depend on the
	level at which the analysis is being performed.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>A key step in
	cladistic analysis is outgroup comparison, which is used to
	differentiate shared primitive characters from shared derived ones.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>To
	do this, we need to identify an </FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>outgroup,</B></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>
	a species or group of species that is closely related to the species
	that we are studying, but known to be less closely related than any
	members of the study group are to each other.</FONT></FONT></FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>To
	study the relationships among an </FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>ingroup</B></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>
	of five vertebrates (a leopard, a turtle, a salamander, a tuna, and
	a lamprey) on a cladogram, an animal called the lancelet is a good
	choice.</FONT></FONT></FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The lancelet is a
	small member of the Phylum Chordata that lacks a backbone.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The species making
	up the ingroup display a mixture of shared primitive and shared
	derived characters.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>In an outgroup
	analysis, the assumption is that any homologies shared by the
	ingroup and outgroup are primitive characters that were present in
	the common ancestor of both groups.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Homologies present
	in some or all of the ingroup taxa are assumed to have evolved after
	the divergence of the ingroup and outgroup taxa.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>In our example, a
	notochord, present in lancelets and in the embryos of the ingroup,
	is a shared primitive character and, thus, not useful for sorting
	out relationships between members of the ingroup.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The presence of a
	vertebral column, shared by all members of the ingroup but not the
	outgroup, is a useful character for the whole ingroup.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The presence of
	jaws, absent in lampreys and present in the other ingroup taxa,
	helps to identify the earliest branch in the vertebrate cladogram.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Analyzing the
	taxonomic distribution of homologies enables us to identify the
	sequence in which derived characters evolved during vertebrate
	phylogeny.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>A cladogram
	presents the chronological sequence of branching during the
	evolutionary history of a set of organisms.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>However, this
	chronology does not indicate the time of origin of the species that
	we are comparing, only the groups to which they belong.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>For example, a
	particular species in an old group may have evolved more recently
	than a second species that belongs to a newer group.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>A
	cladogram is </FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><I>no</I></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>t
	a phylogenetic tree.</FONT></FONT></FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>To convert it to a
	phylogenetic tree, we need more information from sources such as the
	fossil record, which can indicate when and in which groups the
	characters first appeared.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Any chronology
	represented by the branching pattern of a phylogenetic tree is
	relative (earlier versus later) rather than absolute (so many
	millions of years ago).</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Some kinds of tree
	diagrams can be used to provide more specific information about
	timing.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>In
	a </FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>phylogram,</B></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>
	the length of a branch reflects the number of genetic changes that
	have taken place in a particular DNA or RNA sequence in a lineage.</FONT></FONT></FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Even though the
	branches in a phylogram may have different lengths, all the
	different lineages that descend from a common ancestor have survived
	for the same number of years.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Humans and bacteria
	had a common ancestor that lived more than 3 billion years ago.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>This ancestor was a
	single-celled prokaryote and was more like a modern bacterium than
	like a human.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Even though
	bacteria have apparently changed little in structure since that
	common ancestor, there have nonetheless been 3 billion years of
	evolution in both the bacterial and eukaryotic lineages.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>These
	equal amounts of chronological time are represented in an </FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>ultrameric
	tree.</B></FONT></FONT></FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>In an ultrameric
	tree, the branching pattern is the same as in a phylogram, but all
	the branches that can be traced from the common ancestor to the
	present are of equal lengths.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Ultrameric trees do
	not contain the information about different evolutionary rates that
	can be found in phylograms.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>However, they draw
	on data from the fossil record to place certain branch points in the
	context of geological time.</FONT></FONT></P>
</UL>
<P STYLE="margin-top: 0.13in; margin-bottom: 0in; line-height: 0.17in">
 <FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><I><B>The
principles of maximum parsimony and maximum likelihood help
systematists reconstruct phylogeny.</B></I></FONT></FONT></P>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>As available data
	about DNA sequences increase, it becomes more difficult to draw the
	phylogenetic tree that best describes evolutionary history.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>If
	you are analyzing data for 50 species, there are 3 × 10</FONT></FONT><SUP><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>76</FONT></FONT></SUP><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>
	different ways to form a tree.</FONT></FONT></FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>According
	to the principle of </FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>maximum
	parsimony,</B></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>
	we look for the simplest explanation that is consistent with the
	facts.</FONT></FONT></FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>In the case of a
	tree based on morphological characters, the most parsimonious tree
	is the one that requires the fewest evolutionary events to have
	occurred in the form of shared derived characters.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>For phylograms
	based on DNA sequences, the most parsimonious tree requires the
	fewest base changes in DNA.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The
	principle of </FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>maximum
	likelihood</B></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>
	states that, given certain rules about how DNA changes over time, a
	tree should reflect the most likely sequence of evolutionary events.</FONT></FONT></FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Maximum likelihood
	methods are designed to use as much information as possible.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Many computer
	programs have been developed to search for trees that are
	parsimonious and likely:</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	“<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Distance”
	methods minimize the total of all the percentage differences among
	all the sequences.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>More complex
	“character-state” methods minimize the total number of base
	changes or search for the most likely pattern of base changes among
	all the sequences.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Although we can
	never be certain precisely which tree truly reflects phylogeny, if
	they are based on a large amount of accurate data, the various
	methods usually yield similar trees.</FONT></FONT></P>
</UL>
<P STYLE="margin-top: 0.13in; margin-bottom: 0in; line-height: 0.17in">
 <FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><I><B>Phylogenetic
trees are hypotheses.</B></I></FONT></FONT></P>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Any phylogenetic
	tree represents a hypothesis about how the organisms in the tree are
	related.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The best hypothesis
	is the one that best fits all the available data.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>A hypothesis may be
	modified when new evidence compels systematists to revise their
	trees.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Many older
	phylogenetic hypotheses have been changed or rejected since the
	introduction of molecular methods for comparing species and tracing
	phylogeny.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Often, in the
	absence of conflicting information, the most parsimonious tree is
	also the most likely.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Sometimes
	there is compelling evidence that the best hypothesis is </FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><I>not</I></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>
	the most parsimonious.</FONT></FONT></FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Nature does not
	always take the simplest course.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>In some cases, the
	particular morphological or molecular character we are using to sort
	taxa actually did evolve multiple times.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>For example, the
	most parsimonious assumption would be that the four-chambered heart
	evolved only once in an ancestor common to birds and mammals but not
	to lizards, snakes, turtles, and crocodiles.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>But
	abundant evidence indicated that birds and mammals evolved from
	</FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><I>different</I></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>
	reptilian ancestors.</FONT></FONT></FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The hearts of birds
	and mammals develop differently, supporting the hypothesis that they
	evolved independently.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The most
	parsimonious tree is not consistent with the above facts, and must
	be rejected in favor of a less parsimonious tree.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The four-chambered
	hearts of birds and mammals are analogous, not homologous.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Occasionally
	misjudging an analogous similarity in morphology or gene sequence as
	a shared derived homology is less likely to distort a phylogenetic
	tree if several derived characters define each clade in the tree.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The strongest
	phylogenetic hypotheses are those supported by multiple lines of
	molecular and morphological evidence as well as by fossil evidence.</FONT></FONT></P>
</UL>
<P STYLE="margin-top: 0.13in; margin-bottom: 0in; font-style: normal; line-height: 0.17in">
<BR>
</P>
<P STYLE="margin-top: 0.13in; margin-bottom: 0in; font-style: normal; line-height: 0.17in">
<FONT COLOR="#ff0000"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>Concept
25.4 Much of an organism’s evolutionary history is documented in
its genome</B></FONT></FONT></FONT></P>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Molecular
	systematics is a valuable tool for tracing an organism’s
	evolutionary history.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The molecular
	approach helps us to understand phylogenetic relationships that
	cannot be measured by comparative anatomy and other nonmolecular
	methods.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>For example,
	molecular systematics helps us uncover evolutionary relationships
	between groups that have no grounds for morphological comparison,
	such as mammals and bacteria.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Molecular
	systematics enables scientists to compare genetic divergence within
	a species.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Molecular biology
	has helped to extend systematics to evolutionary relationships far
	above and below the species level.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Its findings are
	sometimes inconclusive, as in cases where a number of taxa diverged
	at nearly the same time.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The ability of
	molecular trees to encompass both short and long periods of time is
	based on the fact that different genes evolve at different rates,
	even in the same evolutionary lineage.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>For example, the
	DNA that codes for ribosomal RNA (rRNA) changes relatively slowly,
	so comparisons of DNA sequences in these genes can be used to sort
	out relationships between taxa that diverged hundreds of millions of
	years ago.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>In contrast,
	mitochondrial DNA (mtDNA) evolved relatively recently and can be
	used to explore recent evolutionary events, such as relationships
	between groups within a species.</FONT></FONT></P>
</UL>
<P STYLE="margin-top: 0.13in; margin-bottom: 0in; line-height: 0.17in">
 <FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><I><B>Gene
duplication has provided opportunities for evolutionary change.</B></I></FONT></FONT></P>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Gene duplication
	increases the number of genes in the genome, providing opportunities
	for further evolutionary change.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Gene duplication
	has resulted in gene families, which are groups of related genes
	within an organism’s genome.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Like homologous
	genes in different species, these duplicated genes have a common
	genetic ancestor.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>There are two types
	of homologous genes: orthologous genes and paralogous genes.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The
	term </FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>orthologous</B></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>
	refers to homologous genes that are found in different gene pools
	because of speciation.</FONT></FONT></FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The ß hemoglobin
	genes in humans and mice are orthologous.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>Paralogous</B></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>
	genes result from gene duplication and are found in more than one
	copy in the same genome.</FONT></FONT></FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Olfactory receptor
	genes have undergone many gene duplications in vertebrates.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Humans and mice
	each have huge families of more than 1,000 of these paralogous
	genes.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Now that we have
	compared entire genomes of different organisms, two remarkable facts
	have emerged.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Orthologous genes
	are widespread and can extend over enormous evolutionary distances.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Approximately 99%
	of the genes of humans and mice are demonstrably orthologous, and
	50% of human genes are orthologous with those of yeast.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>All living things
	share many biochemical and development pathways.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The number of genes
	seems not to have increased at the same rate as phenotypic
	complexity.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Humans have only
	five times as many genes as yeast, a simple unicellular eukaryote,
	although we have a large, complex brain and a body that contains
	more than 200 different types of tissues.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Many human genes
	are more versatile than yeast and can carry out a wide variety of
	tasks in various body tissues.</FONT></FONT></P>
</UL>
<P STYLE="margin-top: 0.13in; margin-bottom: 0in; font-style: normal; line-height: 0.17in">
<BR>
</P>
<P STYLE="margin-top: 0.13in; margin-bottom: 0in; font-style: normal; line-height: 0.17in">
<FONT COLOR="#ff0000"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>Concept
25.5 Molecular clocks help track evolutionary time</B></FONT></FONT></FONT></P>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>In the past, the
	timing of evolutionary events has rested primarily on the fossil
	record.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>One of the goals of
	evolutionary biology is to understand the relationships among all
	living organisms, including those for which there is no fossil
	record.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>Molecular
	clocks</B></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>
	serve as yardsticks for measuring the absolute time of evolutionary
	change.</FONT></FONT></FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>They are based on
	the observation that some regions of the genome evolve at constant
	rates.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>For these regions,
	the number of nucleotide substitutions in orthologous genes is
	proportional to the time that has elapsed since the two species last
	shared a common ancestor.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>In the case of
	paralogous genes, the number of substitutions is proportional to the
	time since the genes became duplicated.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>We can calibrate
	the molecular clock of a gene by graphing the number of nucleotide
	differences against the timing of a series of evolutionary branch
	points that are known from the fossil record.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The slope of the
	best line through these points represents the evolution rate of that
	molecular clock.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>This rate can be
	used to estimate the absolute date of evolutionary events that have
	no fossil record.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>No molecular clock
	is completely accurate.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Genes
	that make good molecular clocks have fairly smooth </FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><I>average</I></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>
	rates of change.</FONT></FONT></FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>No
	genes mark time with a precise </FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><I>tick-tock</I></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>
	accuracy in the rate of base changes.</FONT></FONT></FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Over time there may
	be chance deviations above and below the average rate.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Rates of change of
	various genes vary greatly.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Some genes evolve a
	million times faster than others.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The molecular clock
	approach assumes that much of the change in DNA sequences is due to
	genetic drift and is selectively neutral.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The
	</FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><B>neutral
	theory</B></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>
	suggests that much evolutionary change in genes and proteins has no
	effect on fitness and, therefore, is not influenced by Darwinian
	selection.</FONT></FONT></FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Researchers
	supporting this theory point out that many new mutations are harmful
	and are removed quickly.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>However, if most of
	the rest are neutral and have little or no effect on fitness, the
	rate of molecular change should be clocklike in their regularity.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Differences in the
	rates of change of specific genes are a function of the importance
	of the gene.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>If the exact
	sequence of amino acids specified by a gene is essential to
	survival, most mutations will be harmful and will be removed by
	natural selection.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>If the sequence of
	genes is less critical, more mutations will be neutral, and
	mutations will accumulate more rapidly.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Some DNA changes
	are favored by natural selection.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>This leads some
	scientists to question the accuracy and utility of molecular clocks
	for timing evolution.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Evidence
	suggests that almost 50% of the amino acid differences in proteins
	of two </FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><I>Drosophila</I></FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>
	species have resulted from directional natural selection.</FONT></FONT></FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Over very long
	periods of time, fluctuations in the rate of accumulation of
	mutations due to natural selection may even out.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Even genes with
	irregular clocks can mark elapsed time approximately.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Biologists are
	skeptical of conclusions derived from molecular clocks that have
	been extrapolated to time spans beyond the calibration in the fossil
	record</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Few fossils are
	older than 550 million years old.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Estimates for
	evolutionary divergences prior to that time may assume that
	molecular clocks have been constant over billions of years.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Such estimates have
	a high degree of uncertainty.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The molecular clock
	approach has been used to date the jump of the HIV virus from
	related SIV viruses that infect chimpanzees and other primates to
	humans.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The virus has
	spread to humans more than once.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The multiple
	origins of HIV are reflected in the variety of strains of the virus.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>HIV-1 M is the most
	common HIV strain.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Investigators have
	calibrated the molecular clock for the virus by comparing samples of
	the virus collected at various times.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>From their
	analysis, they project that the HIV-1 M strain invaded humans in the
	1930s.</FONT></FONT></P>
</UL>
<P STYLE="margin-top: 0.13in; margin-bottom: 0in; line-height: 0.17in">
 <FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><I><B>There is a
universal tree of life.</B></I></FONT></FONT></P>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The genetic code is
	universal in all forms of life.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>From this,
	researchers infer that all living things have a common ancestor.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Researchers are
	working to link all organisms into a universal tree of life.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Two criteria
	identify regions of DNA that can be used to reconstruct the
	branching pattern of this tree.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>The regions must be
	able to be sequenced.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>They must have
	evolved slowly, so that even distantly related organisms show
	evidence of homologies in these regions.</FONT></FONT></P>
</UL>
<UL>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>rRNA genes, coding
	for the RNA component of ribosomes, meet these criteria.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.08in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Two points have
	emerged from this effort:</FONT></FONT></P>
</UL>
<P STYLE="margin-left: 0.25in; margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>1.</FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><I>	The
tree of life consists of three great domains: Bacteria, Archaea, and
Eukarya.</I></FONT></FONT></FONT></FONT></P>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Most prokaryotes
	belong to Bacteria.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Archaea includes a
	diverse group of prokaryotes that inhabit many different habitats.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Eukarya includes
	all organisms with true nuclei, including many unicellular organisms
	as well as the multicellular kingdoms.</FONT></FONT></P>
</UL>
<P STYLE="margin-left: 0.25in; margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
<FONT FACE="Times, serif"><FONT SIZE=2 STYLE="font-size: 11pt"><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>2.</FONT></FONT><FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4><I>	The
early history of these domains is not yet clear.</I></FONT></FONT></FONT></FONT></P>
<UL>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Early in the
	history of life, there were many interchanges of genes between
	organisms in the different domains.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>One mechanism for
	these interchanges was horizontal gene transfer, in which genes are
	transferred from one genome to another by mechanisms such as
	transposable elements.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>Different organisms
	fused to produce new, hybrid organisms.</FONT></FONT></P>
	<LI><P STYLE="margin-top: 0.04in; margin-bottom: 0in; line-height: 0.17in">
	<FONT FACE="Comic Sans MS, cursive"><FONT SIZE=4>It is likely that
	the first eukaryote arose through fusion between an ancestral
	bacterium and an ancestral archaean.</FONT></FONT></P>
</UL>
<DIV TYPE=FOOTER>
	<P CLASS="western" STYLE="margin-left: 0.06in; margin-top: 0.21in; margin-bottom: 0in">
	<FONT FACE="Arial, sans-serif"><FONT SIZE=2 STYLE="font-size: 9pt">Lecture
	Outline for Campbell/Reece </FONT></FONT><FONT FACE="Arial, sans-serif"><FONT SIZE=2 STYLE="font-size: 9pt"><I>Biology</I></FONT></FONT><FONT FACE="Arial, sans-serif"><FONT SIZE=2 STYLE="font-size: 9pt">,
	7</FONT></FONT><SUP><FONT FACE="Arial, sans-serif"><FONT SIZE=2 STYLE="font-size: 9pt">th</FONT></FONT></SUP><FONT FACE="Arial, sans-serif"><FONT SIZE=2 STYLE="font-size: 9pt">
	Edition, © Pearson Education, Inc. 		25-<SDFIELD TYPE=PAGE SUBTYPE=RANDOM FORMAT=PAGE>13</SDFIELD></FONT></FONT></P>
	<P CLASS="western" STYLE="margin-bottom: 0in"><BR>
	</P>
</DIV>
</BODY>
</HTML>